tensorflow_datasets/video/KTH_human_action.py

```diff
@@ -0,0 +1,67 @@
+import tensorflow_datasets.public_api as tfds
+DESCRIPTION="""\
+The current video database containing six types of human actions (walking, jogging, running, boxing, hand waving and hand clapping) performed several times by 25 subjects in four different scenarios: outdoors s1, outdoors with scale variation s2, outdoors with different clothes s3 and indoors s4 as illustrated below. Currently the database contains 2391 sequences. All sequences were taken over homogeneous backgrounds with a static camera with 25fps frame rate. The sequences were downsampled to the spatial resolution of 160x120 pixels and have a length of four seconds in average.
+"""
+DATA_URL=(" http://www.nada.kth.se/cvap/actions/walking.zip","http://www.nada.kth.se/cvap/actions/jogging.zip","http://www.nada.kth.se/cvap/actions/running.zip",
+            "http://www.nada.kth.se/cvap/actions/boxing.zip","http://www.nada.kth.se/cvap/actions/handwaving.zip","http://www.nada.kth.se/cvap/actions/handclapping.zip")
+FRAMES_PER_SEC= 25
+IMG_SHAPE=(160,120)
+CITATION ="""Schuldt, Laptev and Caputo, Proc. ICPR'04, Cambridge, UK """
```

Comment:
this is an incorrect citation format.
Please look on the "citation" paragraph in add_dataset documentation.